sonar-scanner-engine/src/main/java/org/sonar/scanner/bootstrap/ScannerExtensionDictionnary.java

```diff
@@ -105,17 +104,17 @@ public ScannerExtensionDictionnary(ComponentContainer componentContainer, Defaul
     }
     if (org.sonar.api.batch.Sensor.class.equals(type)) {
       // Retrieve new Sensors and wrap then in SensorWrapper
-      for (Object extension : getExtensions(Sensor.class)) {
-        extension = new SensorWrapper((Sensor) extension, sensorContext, sensorOptimizer);
+      for (Sensor sensor : getExtensions(Sensor.class)) {
+        org.sonar.api.batch.Sensor extension = new SensorWrapper(sensor, sensorContext, sensorOptimizer);
         if (shouldKeep(type, extension, module, matcher)) {
           result.add((T) extension);
         }
       }
     }
     if (org.sonar.api.batch.PostJob.class.equals(type)) {
       // Retrieve new PostJob and wrap then in PostJobWrapper
-      for (Object extension : getExtensions(PostJob.class)) {
-        extension = new PostJobWrapper((PostJob) extension, postJobContext, postJobOptimizer);
+      for (PostJob postJob : getExtensions(PostJob.class)) {
+        org.sonar.api.batch.PostJob extension = new PostJobWrapper(postJob, postJobContext, postJobOptimizer);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "PostJob"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)